src/llmq/utils.cpp

```diff
@@ -660,12 +660,15 @@ const CBlockIndex* V19ActivationIndex(const CBlockIndex* pindex)
 {
     assert(pindex);
 
-    LOCK(cs_llmq_vbc);
-    if (VersionBitsState(pindex, Params().GetConsensus(), Consensus::DEPLOYMENT_V19, llmq_versionbitscache) != ThresholdState::ACTIVE) {
-        return nullptr;
-    }
-    int nHeight = VersionBitsStateSinceHeight(pindex, Params().GetConsensus(), Consensus::DEPLOYMENT_V19, llmq_versionbitscache);
-    return pindex->GetAncestor(nHeight);
+    auto nHeight = [&pindex]() -> std::optional<int> {
+       LOCK(cs_llmq_vbc);
+       if (VersionBitsState(pindex, Params().GetConsensus(), Consensus::DEPLOYMENT_V19, llmq_versionbitscache) != ThresholdState::ACTIVE) {
+           return {};
```

Comment:
`return std::nullopt;` to avoid possible confusion.

Similar backport in bitcoin:
```
    Merge #21498: refactor: return std::nullopt instead of {}
```

Reply:
@PastaPastaPasta and how about this one? :)

Reply:
meh

Reply:
why?

Reply:
Done, although I personally don't really care to return std::nullopt. IMO {} is clear, although I do generally prefer comparing to nullopt instead of the boolean operator